### DISCUSSION TITLE
does there need some update with guide and tests in mshadow?

### DISCUSSION THREAD

**Discussion by XiaotaoChen** [apache/incubator-mxnet#15848] [Q&A]
refer to the issue in mshadow [does there need some update with guide and tests?](https://github.com/dmlc/mshadow/issues/355)
i met the same problem. `basic_stream` aborted with the info:
```shell
[10:34:45] ../mshadow/./stream_gpu-inl.h:82: Default GPU stream was used when MSHADOW_FORCE_STREAM was on
Aborted
```
`config.mk` setting as below:
```shell
USE_CUDA = 1
USE_CUDA_PATH = /usr/local/cuda
```
According to the compile error,  i modified some options of compilation in `Makefile`. Adding `-fopenmp` to `CFLAGS` ,  `-std=c++11` to `NVCCFLAGS` and `basic_stream` to `CUBIN` as below:
```shell
export CFLAGS = -Wall -O3 -std=c++11 -fopenmp -I../ $(MSHADOW_CFLAGS)
export LDFLAGS= -lm $(MSHADOW_LDFLAGS)
export NVCCFLAGS = -O3 -std=c++11 --use_fast_math -ccbin $(CXX) $(MSHADOW_NVCCFLAGS)
CUBIN = basic_stream
```
and modify the code or `newStream` in `basic_stream.cu`
```
  stream<gpu> *sm1 = NewStream<gpu>(false, true, 0);
  Stream<gpu> *sm2 = NewStream<gpu>(false, true, 0);
  res.stream_ = NewStream<gpu>(false, true, 0);
```
@wxywb have you fix this error ?

**Comment by vdantu** [comment]
@mxnet-label-bot add [question]
